### DISCUSSION TITLE
mcu 'mcu': Unknown command: t5uid1_ping

### DISCUSSION THREAD

**Discussion by Newcal** [Desuuuu/klipper#114] [Q&A]
Hello everyone,

First time I post something on GitHub so sorry if there is any typo or rules to operate...

I flashed my cr10s pro v2 and everything works perfectly with Klipper. I flashed the screen and so far so good. I put the section : [t5uid1]
firmware: dgus_reloaded
machine_name: CR-10S Pro V2
volume: 60
brightness: 50
z_min: 0
z_max: 350

And this is what I got now once save and restart : 
mcu 'mcu': Unknown command: t5uid1_ping

This type of error is frequently caused by running an older
version of the firmware on the micro-controller (fix by
recompiling and flashing the firmware).

Known versions: host=dgus-reloaded-211225-0-g4e2090f8, mcu=v0.10.0-320-g7e654aed

Is there any fix ? Thank you :)
JS

**Comment by Desuuuu** [comment]
You need to compile the MCU firmware from my fork and flash it. Make sure to enable the T5UID1 option while configuring the firmware as well.
